base/third_party/go-wrapper/go-wrapper

```diff
@@ -86,7 +86,25 @@ case "$cmd" in
 		;;
 		
 	run)
-		set -x; exec "$goBin" "$@"
+		if [ -n "${CLOUD_DEBUG_ENABLE}" ] && [ "${CLOUD_DEBUG_ENABLE}" != "false" ]; then
```

Comment:
This can be shortened to `[ -n "$CLOUD_DEBUG_ENABLE" && "$CLOUD_DEBUG_ENABLE" != false ]`